src/hooks/useSignMessage/types.ts

```diff
@@ -14,7 +14,8 @@ export type UseSignMessageProps = {
     approveQuickAcc?: (confCode: number) => void
   ) => Promise<any>
   onLastMessageSign: () => void
-  getHardwareWallet: (device?: any) => any
+  getHardwareWallet: (device?: any) => any,
+  connections: any[]
```

Comment:
New prop, `connections`, is added to the hook props. But it's not used anywhere. Is this still needed?